l10n_it_riba/models/account.py

```diff
@@ -58,9 +59,31 @@ def _compute_is_past_due(self):
             if len(invoice.past_due_move_line_ids) != reconciled_past_due:
                 invoice.is_past_due = True
 
+    def _compute_open_amount(self):
+        for invoice in self:
+            if invoice.is_riba_payment:
+                today = fields.Date.today()
+                open_amount_line_ids = invoice.line_ids.filtered(
+                    lambda line, today=today: line.riba
+                    and line.display_type == "payment_term"
+                    and line.date_maturity > today
+                )
+                invoice.open_amount = sum(open_amount_line_ids.mapped("balance"))
+            else:
+                invoice.open_amount = 0.0
+
     riba_credited_ids = fields.One2many(
         "riba.slip", "credit_move_id", "Credited RiBa Slips", readonly=True
     )
+
+    open_amount = fields.Float(
+        digits="Account",
+        compute="_compute_open_amount",
+        store=True,
```

Comment:
L'implementazione dipende da `today` quindi `store=True` non si potrebbe applicare in questo caso